docs/source/claimed_systems.rst

```diff
@@ -0,0 +1,201 @@
+Benchmarking Systems
+####################
+
+Here, we provide the full list of benchmarking systems from the `Schrodinger binding free energy benchmarks <https://github.com/schrodinger/public_binding_free_energy_benchmark/tree/v2.0>`_.
+The claimed systems are the datasets that have been picked by our industrial partners to be prepared and run, while the unclaimed systems will not be included in the benchmarking effort.
+We will update this list if more systems are being claimed.
+
+.. list-table:: List of benchmarking systems
```

Comment:
Not sure if this is wip, but just in case - the ordering has gone a bit off here: https://industrybenchmarks2024--18.org.readthedocs.build/en/18/claimed_systems.html (i.e. some systems are going into the last column)